dashdotdb/controllers/serviceslometrics.py

```diff
@@ -0,0 +1,12 @@
+from dashdotdb.services.serviceslometrics import ServiceSLOMetrics
+
+
+def post(cluster, body):
+    imv = ServiceSLOMetrics(cluster=cluster)
+    imv.insert(slo=body)
+    return 'ok'
+
+
+def search(cluster, namespace, sli_type, name):
+    imv = ServiceSLOMetrics(cluster, namespace, sli_type, name)
+    return imv.get_slometrics()
```

Comment:
Are you also going to expose condensed data in the /metrics endpoint?

Reply:
Good point! I'll do that. ;D